replay/metrics/base_metric.py

```diff
@@ -254,6 +268,16 @@ def _get_metric_distribution(self, recs: DataFrame, k: int) -> DataFrame:
         :param k: depth cut-off
         :return: metric distribution for different cut-offs and users
         """
+        if self._use_scala_udf:
+            # Possibly bad approach to define column names for udf call
```

Comment:
we believe that `recs` dataframe comes from get_enriched_recommendations and has pre-defined structure. better add an order of columns to the docstring in :param recs: recommendations

Reply:
so cols could be columns[1:]